fix(audit-level): add `info` audit level

wraithgar opened the pull request:
This is a valid level but wasn't configured to be allowed.

Also added this param to the usage output for `npm audit`

![Screen Shot 2021-03-23 at 10 30 35 AM](https://user-images.githubusercontent.com/36607/112191046-d5b11280-8bc2-11eb-8b4c-655338f93993.png)

## References
Related to https://github.com/npm/cli/issues/2715

wraithgar committed:
fix(audit-level): add `info` audit level

This is a valid level but wasn't configured to be allowed.

Also added this param to the usage output for `npm audit`

PR-URL: https://github.com/npm/cli/pull/2923
Credit: @wraithgar
Close: #2923
Reviewed-by: @ruyadorno

ruyadorno commented:
I don't think it's related to the changes here but it'd be nice if the helper that generates these options could break/wrap to next line once it hits 80 characters

wraithgar committed:
fix(install): ignore auditLevel

`npm install` should not be affected by the `auditLevel` config, as the
results of audit do not change its exit status.

PR-URL: https://github.com/npm/cli/pull/2929
Credit: @wraithgar
Close: #2929
Reviewed-by: @ljharb, @ruyadorno
